Fushigi/ui/widgets/LevelViewport.cs

```diff
@@ -362,7 +362,7 @@ TileBfresRender CreateTileRendererForSkin(SkinDivision division, string skinName
             //Display skybox
             EnvironmentData.RenderSky(gl, this.Camera);
 
-            foreach (var actor in this.mArea.GetActors())
+            foreach (var actor in this.mArea.GetActors().OrderBy(x => x.mTranslation.Z))
```

Comment:
This code is completely unrelated to the commit, remove or explain
of course you can also put it in a seperate commit, I just feel like this **could** be a huge change in performance and UX so a commit message stating the intention would be very welcome